trac-env/htdocs/css/trachacks.scss

```diff
@@ -274,15 +274,17 @@ div[role="main"]{
       }
     }
 
-    .inlinebuttons input[type="submit"] {
+    .inlinebuttons input[type="submit"],
+    button.trac-revert {
       background: #ddd;
       color: #444;
       font-size: 1.1rem;
-    }
+      height: auto;
```

Comment:
I wonder if `height: initial` gives the same result as "auto"? That seems to match the semantics of what you are after, but whatever works.